components/feature/prompts/src/main/java/mozilla/components/feature/prompts/PromptFeature.kt

```diff
@@ -565,6 +578,16 @@ class PromptFeature private constructor(
     ) {
         // Requests that are handled with dialogs
         val dialog = when (promptRequest) {
+            is SaveCreditCard -> {
+                if (!isCreditCardAutofillEnabled.invoke()) return
```

Comment:
Could we add a test case to validate when the `isCreditCardAutofillEnabled` is `false`? :)